ceph_iscsi_config/lun.py

```diff
@@ -407,6 +407,7 @@ def allocate(self):
                 self.logger.info("rbd image {} resized "
                                  "to {}".format(self.config_key,
                                                 self.size))
+                self.size_bytes = rbd_image._get_size_bytes()
```

Comment:
I think it would be best to just move this to rbd_size() because that function is supposed to sync things up and it already knows the correct size so not need for the extra size() call in _get_size_bytes.

Could you also test/review these patches:

https://github.com/open-iscsi/tcmu-runner/pull/346
https://github.com/ceph/ceph-iscsi-config/pull/27